main.go

```diff
@@ -269,63 +269,130 @@ func getRDSInstances() ([]RDSInfo, error) {
 	}))
 
 	svc := rds.New(sess)
-	input := &rds.DescribeDBInstancesInput{
-		// Supported engine versions are referenced here
-		// https://docs.aws.amazon.com/cli/latest/reference/rds/describe-db-engine-versions.html#options
-		Filters: []*rds.Filter{
-			{
-				Name:   aws.String("engine"),
-				Values: []*string{aws.String("mariadb")},
+	var nextToken *string
+	// input := &rds.DescribeDBInstancesInput{
```

Comment:
Please do not comment out, simply delete the line to make your changes easy to understand.
The commit history will remain in git.